src/services/account_access_definitions.js

```diff
@@ -60,6 +60,21 @@ export default class AccountAccessDefinitions {
     };
   }
 
+  validateAndCastFindAccountParams(params) {
```

Comment:
Maybe we should extract this method to some kind of utils or helpers and parametrize it with an array of allowed parameters? We have a very similar method in entity builder for events, and we are also going to create one more, for assets.